static/templates/compose.hbs

```diff
@@ -105,21 +105,13 @@
                                 <div class="compose_bottom_bottom_container">
                                     <span id="compose_limit_indicator"></span>
                                     <div class="enter_sends">
-                                        <label id="enter-sends-label" class="compose_checkbox_label tippy-zulip-tooltip" data-tippy-content="Click to switch">
-                                            <span class="enter_sends_true">
-                                                <kbd>Enter</kbd> {{t 'to send' }}
-                                            </span>
-                                            <span class="enter_sends_false">
-                                                <kbd>Ctrl</kbd> + <kbd>Enter</kbd> {{t 'to send' }}
-                                            </span>
-                                            <i class="fa fa-exchange" aria-hidden="true"></i>
-                                            <span class="enter_sends_true">
-                                                <kbd>Ctrl</kbd> + <kbd>Enter</kbd> {{t 'to add a new line' }}
-                                            </span>
-                                            <span class="enter_sends_false">
-                                                <kbd>Enter</kbd> {{t 'to add a new line' }}
-                                            </span>
-                                        </label>
+                                        <span class="enter_sends_true tippy-zulip-tooltip" data-tippy-content="{{t 'Switch behaviour' }}">
+                                            <kbd>Enter</kbd> {{t 'to send' }}
+                                        </span>
+                                        <span class="enter_sends_false tippy-zulip-tooltip" data-tippy-content="{{t 'Switch behaviour' }}">
+                                            <kbd>Ctrl</kbd> + <kbd>Enter</kbd> {{t 'to send' }}
```

Comment:
I think we want to use the `z-link` approach or something for these translations, since word order for "{keys} to send" is likely to vary by language, and thus we want that to be the string that translators are translating.